pubsub/system-test/subscriptions.test.js

```diff
@@ -35,39 +36,56 @@ describe('pubsub:subscriptions', function () {
 
   describe('createSubscription', function () {
     it('should create a subscription', function (done) {
-      program.createSubscription(topicName, subscriptionName, function (err, subscription) {
+      program.createSubscription(topicName, subscriptionName, function (err, subscription, apiResponse) {
         assert.ifError(err);
-        assert.equal(subscription.name, name);
+        assert.equal(subscription.name, fullSubscriptionName);
         assert(console.log.calledWith('Created subscription %s to topic %s', subscriptionName, topicName));
-        // The next test sometimes fails, so, slow this test down
+        assert.notEqual(apiResponse, undefined);
+        // Listing is eventually consistent, so give the index time to update
         setTimeout(done, 5000);
       });
     });
   });
 
+  describe('getSubscriptionMetadata', function () {
+    it('should get metadata for a subscription', function (done) {
+      program.getSubscriptionMetadata(subscriptionName, function (err, metadata) {
+        assert.ifError(err);
+        assert.equal(metadata.name, fullSubscriptionName);
+        assert(console.log.calledWith('Got metadata for subscription: %s', subscriptionName));
+        done();
+      });
+    });
+  });
+
   describe('listSubscriptions', function () {
     it('should list subscriptions', function (done) {
       program.listSubscriptions(topicName, function (err, subscriptions) {
```

Comment:
I think we're testing what needs to be tested with both of the list samples. Both hit the api, and we're verifying that both lists contain at least the subscription we created earlier. Depending on the project that running the system tests, there may or may not be more subscriptions in the list. We _could_ create another topic and subscription, just to verify that the second list contains both subscriptions. What do you think?